Makefile

```diff
@@ -30,4 +30,20 @@ start-services: set-linekdpipes-etl-configurations
 
 stop-services:
 	@ echo "$(BUILD_PRINT)Stopping the Docker compose services"
-	@ docker-compose --file docker/docker-compose.yml --env-file docker/.env down
\ No newline at end of file
+	@ docker-compose --file docker/docker-compose.yml --env-file docker/.env down
+
+#-----------------------------------------------------------------------------
+# Template commands
```

Comment:
we need the other two commands as well, don't we?

Reply:
yes, they will be added. I implemented everything for `rdf-differ`, and then I'll replicate the same thing for the other 2